src/icon-tab-bar.tsx

```diff
@@ -97,19 +20,49 @@ type Props = {
 export const IconTabBar: FC<Props> = ({ tabs, selectedTab, onTabSelected }) => {
   const iconTabs = tabs.map(project => {
     const isSelected = project === selectedTab;
-    const classes = (cls: string) =>
-      isSelected ? `${cls} ${cls}--selected` : cls;
+    const buttonClasses = [
+      'min-w-24',
+      'h-12',
+      'border',
+      'rounded-3xl',
+      'px-4',
+      'py-3',
+      'flex',
+      'gap-2',
+      'items-center',
+      'justify-center',
+    ];
+    if (isSelected) {
```

Comment:
I think if you wanted to you could leave the classNames static and use the `aria-selected:` prefix like:

`classNames="cursor-pointer aria-selected:cursor-default text-purple-500 aria-selected:text-white bg-transparent aria-selected:bg-purple-500 border-grey-300 aria-selected:border-purple-500"`

The latter ones override the earlier ones, so there's no "not-aria-selected" prefix.

Reply:
I don't mind doing it this way -- apart from anything else, it's a way to avoid having an absurdly long line of text